uuid.go

```diff
@@ -76,6 +90,17 @@ func Equal(uuid1, uuid2 UUID) bool {
 	return bytes.Equal(uuid1, uuid2)
 }
 
+// Array returns an array representation of uuid that can be used as a map key.
+// Array panics if uuid is not valid.
+func (uuid UUID) Array() Array {
+	if len(uuid) != 16 {
```

Comment:
My reasoning was that an error was too cumbersome. When the primary use of this type is to be used as a map key having a single return value is convenient.

```go
m[u.Array()] = record
```

I figured a panic was better than an zero-values Array because it doesn't mask bugs (getting an array of a nil UUID seems like a bug to me).